website/docs/r/consumption_budget_subscription.html.markdown

```diff
@@ -0,0 +1,178 @@
+---
+subcategory: "Consumption"
```

Comment:
I created a `Consumption` subcategory based on API spec, it could also go under the existing `Cost Management` subcategory as well based on the goal of budgets